SupersonicSound/LowLevel/Sound.cs

```diff
@@ -32,6 +33,24 @@ public static Sound FromFmod(FMOD.Sound sound)
             return new Sound(sound);
         }
 
+        public void Dispose()
```

Comment:
You're right, I didn't see that it's a struct, I just copy/pasted from System ;) Fixed in the next commit.